src/models/Call.ts

```diff
@@ -537,3 +552,253 @@ export class JitsiCall extends Call {
         this.setDisconnected();
     };
 }
+
+interface ElementCallMemberContent {
+    "m.expires_ts": number;
+    "m.calls": {
+        "m.call_id": string;
+        "m.devices": {
+            "m.device_id": string;
+        }[];
+    }[];
+}
+
+/**
+ * A group call using MSC3401 and Element Call as a backend.
+ * (somewhat cheekily named)
+ */
+export class ElementCall extends Call {
+    public static readonly CALL_EVENT_TYPE = new NamespacedValue(null, "org.matrix.msc3401.call");
+    public static readonly MEMBER_EVENT_TYPE = new NamespacedValue(null, "org.matrix.msc3401.call.member");
```

Comment:
I am not sure about that. But I would put the stable name into these as well. So that it works with the stable and unstable names.

Reply:
Right, I know GroupCallUtils had the `m.call` identifiers there as well, but on second thought it felt a bit wrong to include them here, since they don't technically exist in any specified form until MSC3401 is accepted. In the future, once that happens, this will at least make it trivial to add the stable names.